src/control/fault/fault_test.go

```diff
@@ -96,6 +98,12 @@ func TestFaults(t *testing.T) {
 					t.Fatalf("expected %q, got %q", tc.expFaultStr, tc.testErr)
 				}
 			}
+
+			isFault := fault.IsFault(tc.testErr)
+			if tc.expNotFault && isFault {
+				t.Fatalf("expected %+v to not be a fault", tc.testErr)
```

Comment:
if you end up making any updates, how about: t.Fatalf"unexpected fault %+v", tc.testErr)